Library/Homebrew/cmd/update.sh

```diff
@@ -219,8 +219,16 @@ pull() {
 
   if [[ -z "$HOMEBREW_MERGE" ]]
   then
+    if [[ -n "$HOMEBREW_VERBOSE" ]]
+    then
+      echo "Rebasing $DIR onto origin/$UPSTREAM_BRANCH ..."
+    fi
     git rebase "${QUIET_ARGS[@]}" "origin/$UPSTREAM_BRANCH"
   else
+    if [[ -n "$HOMEBREW_VERBOSE" ]]
+    then
+      echo "Merging origin/$UPSTREAM_BRANCH into $DIR..."
```

Comment:
I'm fine with either solution as long as it's consistent and I'm not 100% sure about typographical conventions about this in the English language. Still I'm a tiny bit bothered that the `...` could be misinterpreted to be a part of the directory (as silly as that might sound).